snap/hooks/install

```diff
@@ -30,16 +31,27 @@ fi
 chown -R 584788:root "${SNAP_COMMON}"/*
 
 # Copy over the mongod.conf and create needed directories/files
-echo "configuration file does not exist."
+echo "mongod configuration file does not exist."
 echo "copying default config to ${MONGO_CONFIG_FILE}"
 cp -r ${SNAP}/etc/mongod.conf ${MONGO_CONFIG_FILE}
 
+# Copy over the mongos.conf and create needed directories/files
+echo "mongos configuration file does not exist."
+echo "copying default config to ${MONGOS_CONFIG_FILE}"
+cp -r ${SNAP}/etc/mongod.conf ${MONGOS_CONFIG_FILE}
+
+
 
 # mongod.conf default values are not consistent with the snap directory system.
-sed -i "s/fork: true/fork: false/g" $MONGO_CONFIG_FILE
-sed -i "s:/var/log/mongodb:$LOGS:g" $MONGO_CONFIG_FILE
-sed -i "s:/var/lib/mongodb:$DATA:g" $MONGO_CONFIG_FILE
-sed -i "s:/var/run:/tmp:g" $MONGO_CONFIG_FILE
```

Comment:
noob question (maybe something I missed in yq work) but is an equivalent needed for `sed -i "s:/var/run:/tmp:g" $MONGO_CONFIG_FILE
`